packages/flame/lib/src/components/sprite_animation_component.dart

```diff
@@ -20,22 +20,33 @@ class SpriteAnimationComponent extends PositionComponent
   /// Whether the animation is paused or playing.
   bool playing;
 
+  /// When set to true, the component is auto-resized to match the
+  /// size of current animation sprite.
+  bool _autoResize;
+
   /// Creates a component with an empty animation which can be set later
   SpriteAnimationComponent({
     this.animation,
+    bool? autoResize,
     bool? removeOnFinish,
     bool? playing,
```

Comment:
```suggestion
    bool removeOnFinish = false,
    bool playing = true,
```
These could be set here instead of further down right?
No really related to this PR though.

Reply:
Yes it can be. Fixed the same in `SpriteAnimationGroupComponent` as well.